examples/.experimental/next-app-dir/src/app/alt-server-action/_actions.ts

```diff
@@ -0,0 +1,7 @@
+'use server';
+
+import { api, createAction } from '~/trpc/server';
+
+export const createPostAction = createAction(api.createPost, {
+  revalidates: ['/alt-server-action', api.getPost],
```

Comment:
tags and paths are interopable: https://github.com/vercel/next.js/blob/7dfa56c7141ae2556af0d7fd837e0af686d8fa55/packages/next/src/server/web/spec-extension/revalidate-path.ts#L3

so this should revalidate both the `/alt-server-action` path as well as the `getPost` procedure (although since we're not currently accounting for any inputs (and cacheKeys aren't fuzzy matched) the latter wont work just yet)